examples/statemachine/src/cli/index.ts

```diff
@@ -22,12 +20,12 @@ program
 
 program
     .command('generate')
-    .argument('<file>', `possible file extensions: ${metaData.extensions.join(', ')}`)
+    .argument('<file>', `possible file extensions: ${languageMetaData.fileExtensions.join(', ')}`)
     .option('-d, --destination <dir>', 'destination directory of generating')
     .description('generates a C++ CLI to walk over states')
     .action((fileName: string, opts: GenerateOptions) => {
-        const statemachine = extractAstNode<Statemachine>(fileName, metaData.languageId, metaData.extensions, createStatemachineServices());
-        const generatedFilePath = new StatemachineGenerator(statemachine, fileName, opts.destination).generate();
+        const statemachine = extractAstNode<Statemachine>(fileName, languageMetaData.languageId, languageMetaData.fileExtensions, createStatemachineServices());
+        const generatedFilePath = generateCpp(statemachine, fileName, opts.destination);
         console.log(colors.green('C++ code generated successfully:'), colors.yellow(generatedFilePath));
```

Comment:
Please concatenate these strings into one. The console is not guaranteed to concatenate them, rather it depends heavily on the console implementation. Don't forget to leave a whitespace in the concatenated strings. Also, having yellow colored text in there usually indicates a warning. Perhaps you should change this to white.